src/ui/qgshananewconnectionbase.ui

```diff
@@ -890,6 +890,16 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="chkUseEstimatedMetadata">
+        <property name="toolTip">
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, estimated table metadata will be used if available. For large tables, this avoids slow table loads and potentially expensive computations, but may result in incorrect layer properties such as layer extent. The fast extent estimation is available on HANA Cloud since QRC1/2024 and will be available on HANA On-Premise with SP8 and above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
How about to rephrase the last sentence "is available starting with QRC1/2024 and SP8 in HANA Cloud and HANA On-Premise respectively."?

Reply:
Sounds better, thanks! Done.